src/github.ts

```diff
@@ -153,7 +156,7 @@ export class GitHub {
         baseUrl: this.apiUrl,
         headers: {
           'user-agent': `release-please/${
-            require('../../package.json').version
+            require(resolve('package.json')).version
```

Comment:
oh sweet - does this mean when I setup vscode debugging per [this article](https://medium.com/@benlesh/debugging-typescript-mocha-tests-with-vscode-89310051531) I won't have to go manually change these paths?

Reply:
It would seem so as I was using the VSCode debugger to step through the code. Would it be useful for me to split this change into another PR?

Reply:
@joeldodge79 just landed @safarmer's other PR with the resolve logic, I think we could then just add `tsnode` as a dev dependency, and add an alternate test script (like `test-vs`) for the benefit of VS Code's debugger.

I'd be fine with us checking in configuration in the `.vscode` folder.